# max rate of the planner
RATE: 25
# goal tolerance
XY_GOAL_TOLERANCE: 0.02
YAW_GOAL_TOLERANCE: 0.1
# maximum speed
V_MAX: 0.35
W_MAX: 1.5
# acceleration at beginning of movements
ACCELERATION: 0.5
# goal distance for deceleration
D_DECELERATION: 0.10
# params for accurate movements to water towers or cube heaps
XY_ACCURATE_GOAL_TOLERANCE: 0.01
YAW_ACCURATE_GOAL_TOLERANCE: 0.05
D_ACCURATE_DECELERATION: 0.35
NUM_RANGEFINDERS: 16
COLLISION_STOP_DISTANCE: 150
COLLISION_STOP_NEIGHBOUR_DISTANCE: 50
